src/dependencies/xerces-c/CMakeLists.txt

```diff
@@ -32,10 +32,10 @@ if (WIN32)
     set( _BUILD_DIR Build/Win32/VC${MSVC_VERSION}/${_DEBUG_OR_RELEASE} )
   endif()
 
-  if (MSVC_VERSION GREATER 12)
-    # Only have project files for VS12 and older
-    set(MSVC_VERSION 12)
-  endiF()
+  #if (MSVC_VERSION GREATER 12)
```

Comment:
It actually builds fine with Visual Studio 2017. They probably just don't test with release candidates, so they haven't update their docs. Official release date is Mar 7.

I left the code in but commented out in case we wanted to put it back in for some reason.